prow/repoowners/repoowners.go

```diff
@@ -384,15 +384,20 @@ func (o *RepoOwners) walkFunc(path string, info os.FileInfo, err error) error {
 	}
 	relPathDir := canonicalize(filepath.Dir(relPath))
 
-	simple := &simpleConfig{}
-	if err := yaml.Unmarshal(b, simple); err == nil && !simple.Empty() {
+	simple, err := ParseSimpleConfig(b)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to unmarshal %s into SimpleConfig.", path)
```

Comment:
I'm not sure if we should start logging this as an error, if the file has the full OWNERS config could it fail parsing the simple config or will it always succeed in loading an empty simple config?

Reply:
You are right... the full config doesn't parse as an empty simple config. Therefore we might log an error while it's OK...
I will put the same if/else as in verify-owners and only complain if neither parse.